### ISSUE TITLE
Dataset request: 15-minute UVI

### ISSUE THREAD

**Issue by junjie** [datagovsg/datagovsg-datasets#1053]
### Please provide the name of the dataset or describe the dataset that you're requesting:

Re-requesting a data set that was previously closed as unavailable #181 

NEA has been providing 15-minute average reading of UVI for a while. So I believe this data is available.

### Do you know which agency provides/can provide this data?
*( Refer to this link for a full list of agencies: https://data.gov.sg/organization )*

National Environment Agency

### How frequently would you need this dataset to be updated?
Example: 
- [X] *Test*

- [ ] Annually
- [ ] Monthly
- [ ] Weekly
- [ ] Daily
- [ ] Hourly
- [X] Real-time

### Can you give a short explanation on how you're planning to use this dataset?

Showing near real-time UVI index to users.

**Comment by lennardl** [comment]
Are you referring to this dataset https://data.gov.sg/dataset/ultraviolet-index-uvi ?

**Comment by junjie** [comment]
Yes, but that data set is only providing 1-hour average. NEA's website has 15-minute average readings:

<img width="858" alt="CleanShot 2021-04-15 at 14 21 55@2x" src="https://user-images.githubusercontent.com/524420/114823336-f8b39880-9df5-11eb-9268-4ee2ae32bb46.png">

https://www.nea.gov.sg/weather/ultraviolet-index

**Comment by lennardl** [comment]
@junjie We do not have the 15mins interval data for UV. The realtime datasets are limited to these https://data.gov.sg/dataset/realtime-weather-readings

**Comment by junjie** [comment]
@lennardl I am aware that Data.gov.sg does not currently have the 15-minute interval data, which is why I'm requesting for it. I might have misunderstood the purpose of this issue tracker, or what is termed a dataset request, so please let me know if I have. I thought the purpose is to request for dataset that is not currently available. The last issue I opened #181 was closed as unavailable, which I took it that the relevant agency does not even have the data to provide. But now that NEA has been publishing this data on their own website, this data must be available, but just not made available on Data.gov.sg.
